src/components/Clients/ReferenceCase.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+import FadeIn from '../../animations/FadeIn';
+import Caption from '../../elements/Caption';
+import H2 from '../../elements/H2';
+import P from '../../elements/P';
+import Span from '../../elements/Span';
+import { useViewportSize } from '../../hooks';
+import Section from '../Section';
+
+const EmphasizedH2 = styled(H2)`
```

Comment:
This const is reused in multiple places, maybe we could move it to one place and import it.

Reply:
Moved all EmphasizedH2 that uses the same css to a new element.